packages/mask/src/extension/popups/hook/useFriends.ts

```diff
@@ -0,0 +1,78 @@
+import { useAsyncRetry } from 'react-use'
+import {
+    type ECKeyIdentifier,
+    EMPTY_LIST,
+    type BindingProof,
+    type ProfileInformation,
+    NextIDPlatform,
+} from '@masknet/shared-base'
+import type { AsyncStateRetry } from 'react-use/lib/useAsyncRetry.js'
+import { useCurrentPersona } from '../../../components/DataSource/usePersonaConnectStatus.js'
+import Services from '../../../extension/service.js'
+import { NextIDProof } from '@masknet/web3-providers'
+import { uniqBy } from 'lodash-es'
+
+export type FriendsInformation = ProfileInformation & {
+    profiles: BindingProof[]
+} & { id: string }
```

Comment:
Why not merging them together?

```suggestion
export type FriendsInformation = ProfileInformation & {
    profiles: BindingProof[]
    id: string
}
```